ADVANCED.md

```diff
@@ -0,0 +1,101 @@
+<h1 align='center'>Advanced Usage</h1>
+
+## Compiling Zap From Source
+
+***Note:*** *If you have installation or compilation issues, please file a [Github issue](https://github.com/LN-Zap/zap-desktop/issues) or ping us in [Slack](https://join.slack.com/t/zaphq/shared_invite/enQtMjkyNTAxNDA3MjE2LWE3NGZjZGE5ZmI1NGQ5YTk3MGQzMTdmNDAwYjNhZTJkMWU0ZWZlNzA0MjJiNDBjMzcxYjcyMDMxNWY3OGNhYWQ).*
+
+### Prerequisites
+
+To compile the Zap wallet, you will need:
+
+- **[Node.js version >= 8](https://nodejs.org)** and **[npm version >= 5](https://www.npmjs.com)**
+- **[yarn](https://yarnpkg.com/lang/en/docs/install/)**
+
+### Downloading Zap
+
+After installing the above prerequisites, clone the repo via git:
+
+```bash
+git clone https://github.com/LN-Zap/zap-desktop.git
+```
+
+### Installing Dependencies
+
+Install all the dependencies with yarn + install grpc:
+
+```bash
+cd zap-desktop
+yarn
+npm run install-grpc
+```
+
+## Lightning Network Daemon (lnd)
+
+Zap requires `lnd` but does not store `lnd` binaries in the Github repo.
+
+To setup `lnd` for Zap, you have two options:
+
+- [Light Client](#light-client)
+- [Full Bitcoin Node](#full-bitcoin-node)
+
+### Light Client
+
+`lnd` has a light client called `neutrino` which allows you to run `lnd` without requiring a full node. This is the default configuration for the Zap wallet. To use the light client you can either use the [Lightning Labs Binary](#lightning-labs-binary) or [Compile lnd](#compile-lnd).
```

Comment:
This first sentence should be changed to:

`lnd` allows you to configure your node to use the new light client `neutrino` (BIP 157 & BIP 158) which allows you to run `lnd` without requiring a full node.

Or something like that